### PR TITLE
feat(message-system): update config

### LINKED ISSUE #5445: Update messaging system
We would like to:
1. Remove the phishing warning. We will introduce it again in future if needed.
2. Add a warning about revision `e1c605ca90c6107befe11692b4675b7537f3a658` as in https://github.com/trezor/trezor-suite/pull/5291.

### PR DESCRIPTION
**Pull request by tomasklim**
Close #5445
+ plus https://satoshilabs.slack.com/archives/C03D2GGJ8G0/p1652986720187559?thread_ts=1652983176.673489&cid=C03D2GGJ8G0
+ plus deleted old info about broken auto-update

### CODE REVIEW

**Comment by tsusanka** [comment]
> that versions with broken autoupdate are not used

I see only 22.2.3 in analytics so 👍 .

> new fw revision is model T

You meant model `1` right? :). Let's go ahead with it, see https://github.com/trezor/trezor-suite/issues/5445#issuecomment-1132701696.

File: packages/suite-data/src/message-system/config/config.v1.json [hunk 1]
```diff
@@ -147,10 +71,10 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "22.4.3",
+                        "desktop": "<=22.4.3",
                         "mobile": "!",
-                        "web": "22.4.3",
-                        "revision": "969161f41dcd889fa1fb0cf5e1d3437fddd8c26d"
+                        "web": "<=22.4.3",
```
Comment (tsusanka) [inline packages/suite-data/src/message-system/config/config.v1.json]: Wait, wait, you meant `>=22.4.3`, no?

**Review by tsusanka — CHANGES_REQUESTED** [review]

File: packages/suite-data/src/message-system/config/config.v1.json [hunk 2]
```diff
@@ -147,10 +71,10 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "22.4.3",
+                        "desktop": "<=22.4.3",
```
Comment (matejkriz) [inline packages/suite-data/src/message-system/config/config.v1.json]: This excludes the latest version, right? shouldn't it be the other way around?
```suggestion
                        "desktop": ">=22.4.3",
```

Comment (tomasklim) [reply packages/suite-data/src/message-system/config/config.v1.json]: 🤦

File: packages/suite-data/src/message-system/config/config.v1.json [hunk 3]
```diff
@@ -197,7 +129,7 @@
                 }
             ],
             "message": {
-                "id": "e47cda88-3f39-455b-b4de-01c4270d16e5",
+                "id": "6f793f82-5e13-492f-803c-100c8dcc3cb6",
```
Comment (matejkriz) [inline packages/suite-data/src/message-system/config/config.v1.json:148]: By changing the message.id the message will be displayed again for the users who dismissed it, right? That sounds right to me in this case, just to confirm my understanding...

Comment (tomasklim) [reply packages/suite-data/src/message-system/config/config.v1.json:148]: Yes, exactly.

File: packages/suite-data/src/message-system/config/config.v1.json (hunk 1 above)
Comment (tomasklim) [reply packages/suite-data/src/message-system/config/config.v1.json]: Yes. I spent about 1 minute to make this condition right and in the end, I messed it up 🤦

Comment (tomasklim) [reply packages/suite-data/src/message-system/config/config.v1.json]: Fixed

**Review by tsusanka — APPROVED** [review]
LGTM. I have a small preference to deploy this on Monday, just in case. (and maybe do a quick QA check)

**Comment by tomasklim** [comment]
> LGTM. I have a small preference to deploy this on Monday, just in case. (and maybe do a quick QA check)

I agree. I deployed it to dev for now

**Review by matejkriz — APPROVED** [review]

**Pull request merged**